src/legacy/core_plugins/kibana/public/management/sections/settings/components/component_registry.ts

```diff
@@ -17,7 +17,10 @@
  * under the License.
  */
 
-const registry = {};
+import { FunctionComponent } from 'react';
+
+const registry: { [key: string]: any } = {};
+type Id = string;
```

Comment:
NIT: if Id aliases string, registry should probably be `{ [key: Id]: any }`

Reply:
Addressed.